monitor/sensu/repo.sls

```diff
@@ -7,12 +7,14 @@ sensu-repo:
   pkgrepo:
     - managed
 {% if grains["os_family"] == "Debian" %}
-    - key_url: http://repos.sensuapp.org/apt/pubkey.gpg
-    - name: deb http://repos.sensuapp.org/apt sensu main
+    - key_url: http://repositories.sensuapp.org/apt/pubkey.gpg 
+    - name: deb http://repositories.sensuapp.org/apt sensu main
     - refresh_db: true
 {% elif grains["os_family"] == "RedHat" %}
-    - humanname: sensu-main
-    - baseurl: http://repos.sensuapp.org/yum/el/$releasever/$basearch/
+    - humanname: sensu
+    - name: sensu
```

Comment:
just to follow up on our convo here ... I think if we update this to `name: sensu-repo` it'll "do the right thing" with the existing repo files.